20170701/deux-val-1964-07-29.xml

```diff
@@ -186,7 +186,7 @@
         </HistorySource>
       </MedicationDispensed>
     <!--  <MedicationDispensed>
-        <DrugDescription>butorphanol 10 MG/ML </DrugDescription>
+        <DrugDescription>butorphanol 10 MG/ML</DrugDescription>
```

Comment:
Change looks fine - is this med supposed to be still commented out?

Reply:
Looks like it? It's [part of a larger block comment](https://github.com/uwcirg/pdmp-mock-data/blob/cc8263cdc441810e60d3493b525951e2b2b6509a/20170701/deux-val-1964-07-29.xml#L188-L262) that hasn't otherwise changed.